src/material-experimental/mdc-list/list-base.ts

```diff
@@ -67,7 +69,8 @@ export abstract class MatListItemBase implements AfterContentInit, OnDestroy, Ri
       this._subscriptions.add(this.lines.changes.pipe(startWith(this.lines))
           .subscribe((lines: QueryList<ElementRef<Element>>) => {
             lines.forEach((line: ElementRef<Element>, index: number) => {
-              line.nativeElement.classList.toggle('mdc-list-item__primary-text', index === 0);
+              line.nativeElement.classList
+                  .toggle('mdc-list-item__primary-text', index === 0 && lines.length > 1);
```

Comment:
I know it was like this before, but the second argument to `classList.toggle` actually isn't supported on IE11